cmp/grid/columns/Column.js

```diff
@@ -76,6 +76,7 @@ export class Column {
      *      with ExportFormat.LONG_TEXT to enable text wrapping.
      * @param {(boolean|Column~tooltipFn)} [c.tooltip] - 'true' displays the raw value, or
      *      tool tip function, which is based on AG Grid tooltip callback.
+     * @param {Column~tooltipElementRendererFn} [c.tooltipElementRenderer] - function which returns a React component.
```

Comment:
John -- Bummer that we can't roll this in to `tooltip`?

If we do need a second argument, can we just call it `tooltipElement` and have the element be a `tooltipElementFn` -- seems more analogous to tooltip, and less likely to confuse with the renderer itself.